ios/RIBs/Classes/Router.swift

```diff
@@ -72,11 +72,12 @@ public protocol Routing: RouterScope {
     ///   override in exact `Router` that will handle presentation e.g RootRouter
     ///
     /// - Parameter router: `ViewableRouting` that should be presented
-    func present(_ router: ViewableRouting)
+    /// - Parameter completion: The block to execute after the presentation finishes.
+    func present(_ router: ViewableRouting, completion: (() -> Void)?)
 
     /// Dismisses current presented `ViewableRouting`
     /// - note: This method by defauls calls `present` of it's parent `Routing`
-    func dismiss()
+    func dismiss(completion: (() -> Void)?)
```

Comment:
why we dont use Completable here as a result?
@Granfalloner fyi.

Reply:
Maybe `Single`, in case we opt in to Rx approach.

Reply:
@ValeriyMaliuk

Reply:
@oboje @Granfalloner We can. But should we? At first glance I don't see case when we will benefit from using Rx during presentation or dismissal. Usually they are done from `Router`s and calls from `Interactor` to `Router` is synchronous. Thus if there are no benefits and closure is simpler(no need to import Rx, no `DisposeBag` needed, syntax is cleaner) - closure may be a good choice here.